services/hanke-service/src/integrationTest/kotlin/fi/hel/haitaton/hanke/paatos/PaatosServiceITest.kt

```diff
@@ -100,4 +135,109 @@ class PaatosServiceITest(
             assertThat(result).extracting { it.hakemusId }.containsExactly(hakemus2.id)
         }
     }
+
+    @Nested
+    inner class SaveKaivuilmoituksenPaatos {
```

Comment:
Should there be some unhappy cases too with e.g. failure in getting pdf content from Allu?

Reply:
I added tests for failing the Allu calls.